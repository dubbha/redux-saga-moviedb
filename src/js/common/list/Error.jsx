import React from 'react';

const Error = () => (
  <div className="list__message">
    Error occured
  </div>
);

export default Error;
