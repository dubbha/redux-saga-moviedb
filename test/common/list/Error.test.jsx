import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Error from 'common/list/Error';

describe('Error', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<Error />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
