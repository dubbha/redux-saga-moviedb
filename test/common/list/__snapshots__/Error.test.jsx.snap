// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render successfully 1`] = `
<div
  className="list__message"
>
  Error occured
</div>
`;
